src/script/components/calling/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React, {useState} from 'react';
+
+import {CSSObject} from '@emotion/react';
+
+import {ChevronIcon, IconButton, IconButtonVariant} from '@wireapp/react-ui-kit';
+
+import {handleKeyDown, KEY} from 'Util/KeyboardUtil';
+
+import {
+  chevronLeftStyles,
+  chevronRightStyles,
+  iconButtonStyles,
+  paginationItemsStyles,
+  paginationItemStyles,
+  paginationItemWrapperStyles,
+  paginationWrapperStyles,
+} from './Pagination.styles';
+
+export interface PaginationProps {
+  currentPage: number;
+  totalPages: number;
+  onChangePage: (newPage: number) => void;
+  className?: CSSObject;
+}
+
+const DEFAULT_VISIBLE_DOTS = 5;
+
+const Pagination: React.FC<PaginationProps> = ({totalPages, currentPage, onChangePage, className}) => {
```

Comment:
We not using FC<T> instead of this do like this:
```suggestion
const Pagination = ({totalPages, currentPage, onChangePage, className}: PaginationProps) => {
```